cli/src/main/java/de/jplag/cli/options/CliOptions.java

```diff
@@ -107,6 +107,9 @@ public static class Advanced {
 
         @Option(names = "--log-level", description = "Set the log level for the cli.")
         public Level logLevel = Level.INFO;
+
+        @Option(names = "--include-comments", description = "Enables comment extraction.")
+        public boolean includeComments = false;
```

Comment:
```suggestion
        @Option(names = "--comments", description = "Analyze similarity of comments. This never reduces program similarity.")
        public boolean includeComments = false;
```

Keeps the flag short, but extends the description a bit.

Reply:
Also rename the option to `analyzeComments`. See below.

Reply:
Please also make sure to also extract the new option in the report Viewer.
For that add it to the interface in report-viewer/src/model/CliOptions.ts and add it to be displayed in report-viewer/src/views/InformationView.vue